ow_plexil/src/plans/owlat/OwlatFaultClearDemo.plp

```diff
@@ -0,0 +1,108 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Demonstrates a very simple usage of the FaultClear action, which is
+// used to clear goal errors (and not execution errors).  Continuously
+// monitors for all categories of goal errors, clearing them as they
+// appear.
+
+// An arm goal error is induced and cleared.  The remaining goal error
+// types must be induced manually with the FaultSet action.  This may
+// be done at the Unix command line by entering 'rostopic pub
+// /FaultSet/goal owl_msgs/FaultSetActionGoal' and pressing tab once
+// or twice.  This should print out a template for the message, for
+// which you can edit the 'fault' field and hen press return.  For
+// example, to inject a Task goal error:
+
+// $ rostopic pub /FaultSet/goal owl_msgs/FaultSetActionGoal
```

Comment:
This flags makes it so the rostopic command terminates itself. 
```suggestion
// $ rostopic pub --once /FaultSet/goal owl_msgs/FaultSetActionGoal
```

Reply:
Nice suggestion -- I didn't know about this option.  Added.